frontend/src/components/pages/LoginPage.tsx

```
@@ -47,7 +47,7 @@ const InputField = ({ icon: Icon, ...props }: InputFieldProps) => (
   </div>
 );
 
-export default function LoginPage({ onLogin }: LoginPageProps) {
+export default function LoginPage({ }: LoginPageProps) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)